src/fu-tool.c

```diff
@@ -2736,7 +2736,8 @@ main (int argc, char *argv[])
 		/* set our implemented feature set */
 		fu_engine_request_set_feature_flags (priv->request,
 						     FWUPD_FEATURE_FLAG_DETACH_ACTION |
-						     FWUPD_FEATURE_FLAG_UPDATE_ACTION);
+						     FWUPD_FEATURE_FLAG_UPDATE_ACTION |
+						     FWUPD_FEATURE_FLAG_SWITCH_BRANCH);
```

Comment:
I'm not sure if this should be gated based on interactive or not. Feel free to fix if this is not right.

Reply:
I think @superm1 wanted us to implement the switch-branch warning dialog (with prompt) on upgrade before we added this flag. I can ponder how to do this next week.

Reply:
Yeah

Reply:
implemented same way as fwupdmgr in #2493